docs/source/installing_in_virtualenv.rst

```diff
@@ -0,0 +1,165 @@
+==============================================================
+
+This deployment document assumes that you are running Red Hat Enterprise Linux
+7. Of course, all deployment steps being described through this document also
+apply to other Linux distributions, such as CentOS, openSUSE, or Debian.
+
+Install Apache and prepare a local Nitrate directory
+----------------------------------------------------
+
+First install Apache and mod_wsgi if not present::
+
+    # yum install httpd mod_wsgi
+    # systemctl enable httpd
+    # systemctl start httpd
+
+Next create a directory that will host your Nitrate instance::
+
+    # mkdir /var/www/html/mynitrate
+
+Prepare virtualenv
+------------------
+
+You will install Nitrate inside a virtual environment to avoid conflicts with
+system Python libraries and allow for easier upgrade of dependencies::
+
+    # cd /var/www/html/mynitrate
+    # yum install python-virtualenv
+    # virtualenv venv
+    # ./venv/bin/activate
+
+
+Install Nitrate from source code
+--------------------------------
+
+First install RPM packages which are needed to compile some of the Python
+dependencies. See :doc:`set_dev_env` for more information. Then::
+
+    (venv)# cd /home/<username>/
+    (venv)# git clone https://github.com/Nitrate/Nitrate.git
+    (venv)# cd ./Nitrate/
+    (venv)# git checkout --track [a proper tag or branch]
+    (venv)# pip install -r ./requirements/base.txt
+    (venv)# python setup.py install
+
+.. note::
+
+    Nitrate source code has been cloned into your home directory but
+    has been installed into the virtual environment for Apache!
+
+
+Initialize database
+-------------------
+
+Database is required by Nitrate. Django ORM supports many database backends, but
+for the moment we recommend you to use MySQL because some parts of Nitrate do not
+use the ORM layer but instead hand-crafted SQL queries!
+Create database and user for Nitrate in MySQL::
+
+    mysql> create database nitrate;
+    mysql> GRANT all privileges on nitrate.* to nitrate@'%' identified by 'password';
+
+Configure Nitrate
+-----------------
+
+Create the following files.
+
+
+``/var/www/html/mynitrate/__init__.py`` - empty
+
+``/var/www/html/mynitrate/settings.py``::
+
+    from tcms.settings.product import *
+
+    # SECURITY WARNING: keep the secret key used in production secret!
+    SECRET_KEY = 'top-secret'
+    
+    # SECURITY WARNING: don't run with debug turned on in production!
+    DEBUG = False
+
+    # Database settings
+    DATABASES = {
+        'default': {
+            'ENGINE': 'django.db.backends.mysql',
+            'NAME': 'nitrate',
+            'HOST': '',
+            'USER': 'nitrate',
+            'PASSWORD': 'password',
+        },
+    }
+    # Nitrate defines a 'slave_1' connection
+    DATABASES['slave_1'] = DATABASES['default']
+    
+    STATIC_ROOT = '/var/www/html/mynitrate/static'
+
+
+``/var/www/html/mynitrate/wsgi.py``::
```

Comment:
Nope, this instance of `wsgi.py` at the very least points to the customized `settings.py` file, which inherits from `tcms.settings.product` but allows settings to be overriden and extended.

Depending on the particular environment you may also load the virtualenv from inside wsgi.py and not from Apache. IMO the best practice is to let users have their own configuration depending on the particular instance and not tell them to modify the stock Nitrate settings.py file. This will only result in problems later down the road.